base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/table/ScriptGeneratorAction.java

```diff
@@ -0,0 +1,69 @@
+package uk.ac.stfc.isis.ibex.scriptgenerator.table;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+import uk.ac.stfc.isis.ibex.model.ModelObject;
+
+/**
+ * Class defines one action or 'step' in the script
```

Comment:
End with a full stop otherwise checkstyle complains, stupid I know. You should double check the other docstrings you've added too

Reply:
Full stop has been added